www/install/php/Update-22.04.0-beta.1.php

```diff
@@ -135,6 +135,61 @@
         ADD `login_attempts` INT(11) UNSIGNED DEFAULT NULL,
         ADD `blocking_time` BIGINT(20) UNSIGNED DEFAULT NULL"
     );
+
+    /**
+     * Add new UnifiedSQl broker output
+     */
+    $errorMessage = 'Unable to update cb_type';
+
+    $statement = $pearDB->query(
+        "UPDATE `cb_type` set type_name = 'Perfdata Generator (Centreon Storage) - DEPRECATED'
+            WHERE type_shortname = 'storage'"
+    );
+    $statement = $pearDB->query(
+        "UPDATE `cb_type` set type_name = 'Broker SQL database - DEPRECATED'
+            WHERE type_shortname = 'sql'"
+    );
+
+    $statement = $pearDB->query(
+        "SELECT cb_module_id FROM cb_module
+            WHERE name = 'Storage'"
+    );
+    $moduleId = $statement->fetch();
```

Comment:
even if it should produce, you need to manage the case where fetch returns false